Update to CurseForge 1.19 and change copper block

MMDDKK6500 opened the pull request:
I saw that the copper ores looked alot like iron ores, so i decided to change it to look more like the actual copper ore from vanilla minecraft, in the process i saw that this repository is not up to date with CurseForge, i don't know if it is because in CurseForge it is still in alpha but i decided to try to put it here, I would recommend creating a branch for alpha and put the alpha resource pack in it for now or something, but you decide it

NEW TEXTURE(I also changed the deepslate one)
![copper_ore](https://user-images.githubusercontent.com/34542949/181776243-5964872d-6cb9-4c18-a800-04e5e0ab7698.png)

MMDDKK6500 commented:
> I love this! Would like to see this merged soon :D

Yes! I mean, it's just the version that was uploded to curseforge and I updated the copper ore so that it actually looks like copper.
BUT I think the author of the pack won't accept the pull request.. he just disappared?

federicotorrielli commented:
> > I love this! Would like to see this merged soon :D
> 
> Yes! I mean, it's just the version that was uploded to curseforge and I updated the copper ore so that it actually looks like copper. BUT I think the author of the pack won't accept the pull request.. he just disappared? I'll wait a few more months and if he doesn't do anything I'll see if I can publish the pack with the modification I made

Why wait for months? If it does not respond, we can use your fork and publish it right away. I can help with few other textures if they are missing.

MMDDKK6500 commented:
> > > I love this! Would like to see this merged soon :D
> > 
> > 
> > Yes! I mean, it's just the version that was uploded to curseforge and I updated the copper ore so that it actually looks like copper. BUT I think the author of the pack won't accept the pull request.. he just disappared? I'll wait a few more months and if he doesn't do anything I'll see if I can publish the pack with the modification I made
> 
> Why wait for months? If he does not respond, we can use your fork and publish it right away. I can help with few other textures if they are missing.

That would be aweasome! 

> > > I love this! Would like to see this merged soon :D
> > 
> > 
> > Yes! I mean, it's just the version that was uploded to curseforge and I updated the copper ore so that it actually looks like copper. BUT I think the author of the pack won't accept the pull request.. he just disappared? I'll wait a few more months and if he doesn't do anything I'll see if I can publish the pack with the modification I made
> 
> Why wait for months? If he does not respond, we can use your fork and publish it right away. I can help with few other textures if they are missing.

Well.. he just updated the pack on CurseForge so im not really sure if it would be ok to create a fork and keep publishing, but if you want to try then sure, I can setup a CurseForge project and we put our fork there

federicotorrielli commented:
> > Why wait for months? If he does not respond, we can use your fork and publish it right away. I can help with few other textures if they are missing.
> 
> Well.. he just updated the pack on CurseForge so im not really sure if it would be ok to create a fork and keep publishing, but if you want to try then sure, I can setup a CurseForge project and we put our fork there

That would be awesome!
